Sources/Amplitude/Configuration.swift

```diff
@@ -57,7 +57,8 @@ public class Configuration {
         enableCoppaControl: Bool = false,
         flushEventsOnClose: Bool = true,
         minTimeBetweenSessionsMillis: Int = Constants.Configuration.MIN_TIME_BETWEEN_SESSIONS_MILLIS,
-        trackingSessionEvents: Bool = true,
+        // `trackingSessionEvents` has been replaced by `defaultTracking.sessions`
```

Comment:
Can we add a @Deprecate note for a parameter in an object?

Reply:
No, we can't. `@deprecate` is not supported for a parameter.